source/plugin/data/serialization.rst

```diff
@@ -123,3 +125,56 @@ from persistent files using the :doc:`Configurate Library <../configuration/inde
         final DataView container = translator.translate(node);
         return deserializeHealth(container);
     }
+
+
+DataFormat
+==========
+A alternative to DataTranslators is using :javadoc:`DataFormat` which allows you to convert a data container this allows you to store a data container as a hocon, json or even NBT you can of course retrive data using data format.
+
+This is very useful if your for example using a database to store information as you can then serialize any data container to for example json format
```

Comment:
your -> you are (or you're)
you can use e.g. instead of "for example"
it needs *at least* one comma, preferably after "information"